### PR TITLE
analog: add dataSegment, writeImage commands

### PR DESCRIPTION
**Pull request by DeadByElpy**
fix other commands

### CODE REVIEW

File: src/analog/commands/uplink/getBatteryStatus.ts [hunk 1]
```diff
@@ -59,9 +59,9 @@ export interface IGetBatteryStatusResponse {
 }
 
 
-export const id = 0x051f;
-export const name: string = 'getBatteryStatus';
-export const headerSize: number = 3;
+export const id: types.TCommandId = 0x051f;
```
Comment (DarkPark) [inline src/analog/commands/uplink/getBatteryStatus.ts:62]: please add the same to setParameter

File: src/analog/commands/uplink/writeImage.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+};
+
+
+/**
+ * Convert command parameters to JSON.
+ *
+ * @param parameters - command payload
+ * @returns JSON representation of command parameters
+ */
+export const toJson = ( parameters: IWriteImageParameters ): string => JSON.stringify(parameters);
```
Comment (DarkPark) [inline src/analog/commands/uplink/writeImage.ts:92]: do we need this?

Comment (DeadByElpy) [reply src/analog/commands/uplink/writeImage.ts:92]: it was in source

File: src/analog/commands/downlink/writeImage.ts [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+ *
+ * @packageDocumentation
+ *
+ * @example
+ * ```js
+ * import * as writeImage from 'jooby-codec/analog/commands/downlink/writeImage.js';
+ *
+ * const parameters = {
+ *     offset: 64,
+ *     data: new Uint8Array([
```
[the remark this answers is not in the document]
Comment (DeadByElpy) [reply src/analog/commands/downlink/writeImage.ts]: fixed

File: src/analog/commands/downlink/dataSegment.ts [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    'DataSegment request': {
+        id,
+        name,
+        headerSize,
+        parameters: {
+            segmentationSessionId: 2,
+            segmentIndex: 3,
+            segmentsNumber: 5,
+            isLast: false,
+            data: getBytesFromHex('00 01 02 03 04')
```
[the remark this answers is not in the document]
Comment (DeadByElpy) [reply src/analog/commands/downlink/dataSegment.ts]: fixed

File: src/analog/commands/downlink/index.ts [hunk 5]
```diff
@@ -24,8 +25,8 @@ export * as getTime2000 from './getTime2000.js';
 export * as setParameter from './setParameter.js';
 export * as setTime2000 from './setTime2000.js';
 export * as softRestart from './softRestart.js';
+export * as writeImage from './writeImage.js';
 export * as updateRun from './updateRun.js';
 
 // export {default as DataSegment} from './DataSegment.js';
```
[the remark this answers is not in the document]
Comment (DeadByElpy) [reply src/analog/commands/downlink/index.ts]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**